cookie-banner-rules-list.json

```diff
@@ -893,6 +893,1279 @@
+        ],
+        "optOut": [
+          {
+            "name": "SOCS",
+            "value": "CAESHAgBEhJnd3NfMjAyMjA5MjktMF9SQzEaAnJvIAEaBgiAkvOZBg"
+          }
+        ]
+      },
+      "id": "4a37ab17-dbd7-45cf-83d8-42b84af9a0df"
+    },
+    {
+      "click": {
+        "optIn": "button#L2AGLb",
+        "optOut": "button#W0wltc",
+        "presence": "div.spoKVd"
+      },
+      "domain": "google.be",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "NID",
+            "value": "511=t9ormuzi4e550lBbL3_39XCB_A4vqcJ_loHNIPgMhaj1ThpizkGcTJVeDj6s0U0lmn0OSFYv0aFoXNeZiTTNPVS3IZNv6rv1viywhDkT5Z8tLLm5vSYgniBSKZV9FWU4yoeeDAaEkHkv9r9Lb_MPieXpU1VqtZ9he04yGaww6yA"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "SOCS",
+            "value": "CAESHAgBEhJnd3NfMjAyMjA5MjktMF9SQzEaAnJvIAEaBgiAkvOZBg"
+          }
+        ]
+      },
+      "id": "155e345a-0391-4148-98ba-9a409bc8063f"
+    },
+    {
+      "click": {
+        "optIn": "button#didomi-notice-agree-button",
+        "presence": "div#buttons"
+      },
+      "domain": "orf.at",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "euconsent-v2",
+            "value": "CPgX9oAPgX9oAAHABBENCjCgAPLAAAAAAACYHVQIQADAAVABBACcAKAAWAAyABpAEQARoAmgCeAH4AQIAhABUADVAIQARMAiwBOAC6gGBAMUAfYA_QCCQEagJaAV-AvMBjIDGwGWAOagdSB1QAAACQkAGAAIJBDoAMAAQSCJQAYAAgkEGgAwABBIIUABgACCQRSADAAEEgiAAGAAIJBCIAMAAQSCGAAYAAgkEAAA.flgAAAAAAAAA"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "euconsent-v2",
+            "value": "CPgX9oAPgX9oAAHABBENCjCgAAAAAAAAAACYAAAAAAEhIAMAAQSCHQAYAAgkESgAwABBIINABgACCQQoADAAEEgikAGAAIJBEAAMAAQSCEQAYAAgkEMAAwABBIIA.YAAAAAAAAAAA"
+          }
+        ]
+      },
+      "id": "55ce89fe-e609-46b4-85db-be8a040bf41d"
+    },
+    {
+      "click": {
+        "optIn": "button#didomi-notice-agree-button",
+        "presence": "div#buttons"
+      },
+      "domain": "abv.bg",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "euconsent-v2",
+            "value": "CPgX9oAPgX9oAAHABBENCjCsAP_AAE7AAAIwGMwFgAFAANAAgABUADoAIAAVAAyABoAEUALYAYQBCAFIATSAo8BUgC4QFygLpAXmAxkC84A4AFQAQAAyABoAEUAQgC8wAg0AGAAIJBCIAMAAQSCA.f_gACdgAAAAA"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "euconsent-v2",
+            "value": "CPgX9oAPgX9oAAHABBENCjCgAAAAAAAAAAIwAAAAAABBoAMAAQSCEQAYAAgkEAAA.YAAAAAAAAAAA"
+          }
+        ]
+      },
+      "id": "8d7e2b6c-b276-410a-838f-3685b4fe7cac"
+    },
+    {
+      "click": {
+        "optIn": "button.submitAll",
+        "presence": "div.submitButton"
+      },
+      "domain": "dr.dk",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "CookieConsent",
+            "value": "{stamp:%27WCJvxz/hqBZ/ELOL5sat3WmYnBbYDKFGWYlhItSfcWe2HkSYX3Fi8g==%27%2Cnecessary:true%2Cpreferences:false%2Cstatistics:false%2Cmarketing:false%2Cver:5%2Cutc:1664957292568%2Cregion:%27ro%27}"
```

Comment:
Is this really the opt-in cookie? It looks very similar to the opt-out one. Looks like `statistics` and `marketing` are set to `false`.

Reply:
Updated the rule with the correct opt-in cookie.